draft-ietf-tls-tls13.md

```diff
@@ -2496,6 +2496,22 @@ different curves for (i) the ECDSA or EdDSA key in its certificate,
 and (ii) the ephemeral ECDH key in its "key_share" extension.  The server
 must consider the supported groups in both cases.
 
+Servers SHOULD attempt to select groups for key exchanges that have
+an estimated security level comparable to that of the negotiated
+symmetric bulk cipher key size. The RECOMMENDED groups are as follows:
+
+  * 128-bit level:
+    X25519, secp256r1, ffdhe3072, ffdhe4096
+
+  * 192/256-bit level:
+    X448, secp384r1, secp521r1, ffdhe8192
+
+The estimated symmetric-equivalent strength of ffdhe2048 is only 103-bit,
+and thus MUST NOT be negotiated for TLS 1.3 or later, however it MAY
+be offered for use with prior TLS versions supporting this extension.
```

Comment:
If I remember correctly, this sentence was added due to a statement made by the FFDHE draft author about 2048. I can bring this to the list after a rewrite, at some point.